app/views/layouts/angular-bootstrap/_auth_credentials_angular_bootstrap.html.haml

```diff
@@ -0,0 +1,54 @@
+- validate_url ||= 'log_depot_edit'
+
+%div{"ng-show" => "#{ng_show}"}
```

Comment:
The haml is specific to angular bootstrap (because of the bootstrap classes being used) but not the ```"#{ng_show}"``` usage per se.
This is a shared haml that will be called from other places as well.
The ```ng_show``` local which would have a value of ```true``` or ```false``` would ultimately decide if we are rendering the credentials form input fields or not. 
For e.g. if we are dealing with a ```Database backup``` of type ```Samba```, then ```ng_show``` will be ```true``` and the rest of the form would be rendered.